src/legacy/core_plugins/console/public/quarantined/src/sense_editor/mode/elasticsearch_sql_highlight_rules.ts

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import ace from 'brace';
+
+const { TextHighlightRules } = ace.acequire('ace/mode/text_highlight_rules');
+const oop = ace.acequire('ace/lib/oop');
+
+export const ElasticsearchSqlHighlightRules = function(this: any) {
+  const keywords =
+    'describe|between|in|like|not|and|or|desc|select|from|where|having|group|by|order' +
```

Comment:
How did you generate these strings? If they're drawn from docs, it might be useful to add the link(s) in a comment somewhere.